src/sentry/models/group.py

```diff
@@ -158,24 +156,13 @@ def from_event_id(self, project, event_id):
         Resolves the 32 character event_id string into
         a Group for which it is found.
         """
-        from sentry.models import EventMapping, Event
+        from sentry.models import SnubaEvent
         group_id = None
 
-        # Look up event_id in both Event and EventMapping,
-        # and bail when it matches one of them, prioritizing
-        # Event since it contains more history.
-        for model in Event, EventMapping:
-            try:
-                group_id = model.objects.filter(
-                    project_id=project.id,
-                    event_id=event_id,
-                ).values_list('group_id', flat=True)[0]
-
-                # It's possible that group_id is NULL
-                if group_id is not None:
-                    break
-            except IndexError:
-                pass
+        event = SnubaEvent.objects.from_event_id(event_id, project.id)
```

Comment:
Before merging this, could you please check who is using the method and ensure we won't increase the load on snuba too much. Which means ensure this does not happen in the ingestion flow.

Reply:
This one seems fine

Reply:
Could you please provide some details about the research you have done ? What did you look into ?

Reply:
Yeah sorry, this function seems to be just used in these 3 places, none of them are related to ingestion

```
src/sentry/web/frontend/error_page_embed.py:                    report.group = Group.objects.from_event_id(report.project, report.event_id)
src/sentry/api/endpoints/project_group_index.py:                    matching_group = Group.objects.from_event_id(project, event_id)
src/sentry/api/endpoints/project_user_reports.py:                report.group = Group.objects.from_event_id(project, report.event_id)
```